src/csv_reader/csv_reader.c

```
@@ -263,8 +278,10 @@ void CSVReader_Free
 (
 	CSVReader reader  // CSV reader to free
 ) {
-	// close input file
-	fclose(reader->file);
+	ASSERT(reader != NULL);
```

cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)